### DISCUSSION TITLE
Email Scanner

### DISCUSSION THREAD

**Discussion by drakub** [YetiForceCompany/YetiForceCRM#14526] [General] [answered]
I have configured the mail scanner and I am able to receive emails on the configured SMTP mail. But the challenge I face is that, each and every time the mail is received, I have to manually download it and create a ticket from it. Is there a way like setting a workflow to scan and download the emails and then automatically convert them into tickets. I have got the cron job running so automating at intervals would not be a great deal. All I need is to automatically download these emails, create a ticket and the rest is taken care of the other work flows.

**Comment by drakub** [comment, marked as answer]
Got a work around already

**Reply by mariuszkrzaczkowski** [reply]
Share with everyone what you've set up

**Reply by drakub** [reply]
I just started the cron jobs and left them running
